be/src/vec/common/arena.h

```diff
@@ -271,9 +276,17 @@ class Arena : private boost::noncopyable {
         return res;
     }
 
+    void clear() {
+        delete head;
+        head = new Chunk(initial_size, nullptr), size_in_bytes = head->size(),
+        size_in_allocated = 0;
+    }
+
     /// Size of chunks in bytes.
     size_t size() const { return size_in_bytes; }
 
+    size_t allocated_size() const { return size_in_allocated + head->remaining(); }
```

Comment:
rename this method to:  used_size()  because allocated size maybe allocated but not used.